### PR TITLE
Exclude test data from tarball

### PR DESCRIPTION
**Pull request by CillianFn**
#70 
Verified this works locally using `python setup.py sdist` in both `weather_mv` and `weather_sp`

### CODE REVIEW

**Review by alxmrs — APPROVED** [review]
LGTM! Thanks for the fix.

**Comment by CillianFn** [comment]
@alxmrs np 😄 you asked on another PR, I don't have merge rights so I can't merge this.

**Pull request merged**
